packages/salesforcedx-utils-vscode/src/telemetry/telemetry.ts

```diff
@@ -77,15 +77,19 @@ export class TelemetryService {
    * @param extensionName extension name
    */
   public async initializeService(
-    extensionContext: ExtensionContext,
-    extensionName: string,
-    aiKey: string,
-    version: string
+    extensionContext: ExtensionContext
```

Comment:
As Gordon suggested, we removed other params and only kept the extensionContext. In this way, the EFD extension is not required to be updated, and can still call this function as the extra params will be ignored. This can avoid "users have to update the core extension" issue.